### ISSUE TITLE
Feedback buttons violates WCAG 1.4.3

### ISSUE THREAD

**Issue by auchenberg** [microsoft/vscode-docs#2619]
color-contrast: Elements must have sufficient color contrast (2) WCAG 1.4.3

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI
- https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Code path:
```

<button value="0"><!-- react-text: 10 -->No<!-- /react-text --><span class="sr-only"> , this page was not helpful</span></button>

<button value="1"><!-- react-text: 7 -->Yes<!-- /react-text --><span class="sr-only"> , this page was helpful</span></button>

```

**Comment by Nikhitha-13** [comment]
![Bug_#2619_Color contrast issue](https://user-images.githubusercontent.com/50437925/57441788-46b5de00-7268-11e9-9af1-5b1c924ada90.png)

**Comment by Nikhitha-13** [comment]
**NOTE:**
The issue is observed through out the application in the following URLS:
1. code.visualstudio.com/docs
2.code.visualstudio.com/docs/supporting/faq.
3. code.visualstudio.com/docs/setup/linux.
4. code.visualstudio.com/docs/python/python-tutorial
5. code.visualstudio.com/updates/v1_30
6. code.visualstudio.com/docs/editor/whyvscode
7. code.visualstudio.com/docs/getstarted/tips-and-tricks
8.code.visualstudio.com/updates/v1_29
9.code.visualstudio.com/docs/editor/integrated-terminal
10.code.visualstudio.com/docs/editor/debugging
11.code.visualstudio.com/docs/editor/versioncontrol
12.code.visualstudio.com/docs/getstarted/settings
![Bug_#2619(color contrast keroes)](https://user-images.githubusercontent.com/50437925/57445465-0b1f1200-7270-11e9-8c05-18d56e1b945b.png)
![Bug_#2619_1 3 1 color contrast issue](https://user-images.githubusercontent.com/50437925/57445504-1eca7880-7270-11e9-90e5-fc3f121b0682.png)
![Bug_#2619_1 4 3 issue](https://user-images.githubusercontent.com/50437925/57445539-330e7580-7270-11e9-90d1-41ad65160ea1.png)
![Bug_#2619_keroes color contrast](https://user-images.githubusercontent.com/50437925/57445637-68b35e80-7270-11e9-8487-34143871720d.png)
![Bug_#2619_color contrast](https://user-images.githubusercontent.com/50437925/57445672-808ae280-7270-11e9-9a34-84cf3f6c1980.png)
![Bug_#2619_color contrast (keroes)](https://user-images.githubusercontent.com/50437925/57445737-9d271a80-7270-11e9-87d6-eb89d5b855aa.png)
![Bug_#2619_issue (keroes)(1 3 1)](https://user-images.githubusercontent.com/50437925/57445779-b8922580-7270-11e9-9c70-759a65af292b.png)
![Bug_#2619_Color contrast issue](https://user-images.githubusercontent.com/50437925/57446099-8503cb00-7271-11e9-86bd-96ad9a5236da.png)
![Bug_#2619_color contrast issue](https://user-images.githubusercontent.com/50437925/57446125-96e56e00-7271-11e9-9fba-0910f4bd3795.png)
![Bug_32619_keroes color contrast issue](https://user-images.githubusercontent.com/50437925/57446148-a95fa780-7271-11e9-906c-58c397955166.png)
![Bug_#2619_color contrast](https://user-images.githubusercontent.com/50437925/57446256-ee83d980-7271-11e9-8cc3-8fea998ae9af.png)

**Comment by misolori** [comment]
@auchenberg @Nikhitha-13 I think the tool is calculating the color wrong here? The colors I'm seeing (see below) do not match the colors from the test:

![image](https://user-images.githubusercontent.com/35271042/59224594-a5ca8380-8b83-11e9-8a10-eaae529b3e1a.png)

The test has this button as `#847cb7` but I'm getting `#6D6AA1`.

If you run the tool [using this button](https://s.codepen.io/miguelsolorio/debug/QXWwmW) (exact same styles) then it passes.

**Comment by auchenberg** [comment]
@misolori It seems to be the transparency on the element with `rgba(55,50,119,0.7)` that makes us slip down the below the AAA requirement.

<img width="808" alt="Screen Shot 2019-06-11 at 1 30 45 PM" src="https://user-images.githubusercontent.com/173559/59268631-6f593b00-8c4d-11e9-899c-88666201177f.png"> per https://webaim.org/resources/contrastchecker/


How about changing the bg color to `#373277` which is the one we use for the sidebar links to the left?

<img width="755" alt="Screen Shot 2019-06-11 at 1 31 52 PM" src="https://user-images.githubusercontent.com/173559/59268660-7f711a80-8c4d-11e9-80a1-abc4fedcdb39.png">

**Comment by Nikhitha-13** [comment]
Checked the issue by running keros . The issue is now fixed.

**Comment by fsteffi** [comment]
**GitHubTags:**#CAITestPass_June’19;
